.devcontainer/Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN \
     && go get gotest.tools/gotestsum \
     # --> Go symbols and outline for go to symbol support and test support 
     && go get github.com/acroca/go-symbols@v0.1.1 && go get github.com/ramya-rao-a/go-outline@7182a932836a71948db4a81991a494751eccfe77 \
-    # --> GolangCI-lint
-    && go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.25.0 \
+    # --> Static checker
+    && go install honnef.co/go/tools/cmd/staticcheck@latest \
```

Comment:
in the push hook we have just `go install honnef.co/go/tools/cmd/staticcheck` -  that this version was reason for failure

Reply:
ah! yes, i'll change it in a separate PR